doc/source/tune/api_docs/env.rst

```diff
@@ -49,6 +49,12 @@ These are the environment variables Ray Tune currently considers:
 * **TUNE_MAX_PENDING_TRIALS_PG**: Maximum number of pending trials when placement groups are used. Defaults
   to ``auto``, which will be updated to ``max(16, cluster_cpus * 1.1)`` for random/grid search and ``1``
   for any other search algorithms.
+* **TUNE_NODE_SYNCING_MIN_ITER_THRESHOLD**: When syncing trial data between nodes, only sync if this many
```

Comment:
mention the motivation behind this?
e.g. to mitigate excessive syncing for a lot of short-lived trials.